### PR TITLE
Make the tests flag work with strings with newlines

### PR DESCRIPTION
**Pull request by Lyxal**

### CODE REVIEW

File: vyxal/main.py [hunk 1]
```diff
@@ -96,9 +96,8 @@ def execute_vyxal(file_name, flags, inputs, output_var=None, online_mode=False):
                 inps = [vyxalify(x) for x in inps]
             except:
                 inps = inp.split(", ")
-            temp_ctx = Context()
-            temp_ctx.vyxal_lists = False
-            repred_inps = [vy_repr(x, temp_ctx) for x in inps]
+            repred_inps = [repr(x) for x in inps]
+            print("\n".join(repred_inps), "INPS =", inps, repred_inps)
```
Comment (Steffan153) [inline vyxal/main.py:100]:
```suggestion
```
No need for old debugging messages clogging up the console.

**Review by Steffan153 — DISMISSED** [review]
We should add some tests for this whole test-case thing.

**Comment by Steffan153** [comment]
TESTS WHEN

**Comment by chunkybanana** [comment]
^

**Review by Steffan153 — APPROVED** [review]

**Pull request merged**